xrpl/asyncio/transaction/main.py

```
@@ -583,6 +583,29 @@ async def _calculate_fee_per_transaction_type(
             )
         base_fee += net_fee * counterparty_signers_count
 
+    # SponsorSignature with multi-sign
+    # Fee = (1 + |tx.Signers| + |SponsorSignature.Signers|) × base
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)